### PR TITLE
Bottom-up engraving on Epilog

### PR DESCRIPTION
**Pull request by buxtronix**

### CODE REVIEW

**Comment by t-oster** [comment]
Nice Work.we have to make sure the existing saved settings keep working though. Can you test this?

**Comment by mgmax** [comment]
Looks good:
- Existing settings are loaded correctly (bottom up = false).
- Saving and loading settings works correctly.
- Output file identical to old version if bottom-up = false.

I couldn't test it on a real lasercutter.

One minor note: For LAOS lasers, the property is called "engrave bottom up"; for Epilog it is called "bottom up". While it would be preferable to have both named the same, it won't cause any compatibility problems if we change this later (or even merge the code at one day). This is because It doesn't affect saving the settings to disk or loading them from a file: here the variable name is the same, "engraveBottomUp".

**Comment by buxtronix** [comment]
Note there is one minor change to the Visicut codebase required (LaserPropertyManager) but that is mostly cosmetic to avoid console warnings and is a separate PR.

My code has been tested on an Epilog Mini24, and I also compared the generated PCL to that produced by Epilog's own software.

I elected for just 'bottom up' in the label, as the 'engrave' word is redundant given the operation type anyway.

File: src/com/t_oster/liblasercut/drivers/EpilogCutter.java [hunk 1]
```diff
@@ -452,7 +452,8 @@ public void encode(List<Byte> line, List<Byte> result)
     PrintStream out = new PrintStream(result, true, "US-ASCII");
     if (rp != null)
     {
-      PowerSpeedFocusProperty prop = (PowerSpeedFocusProperty) rp.getLaserProperty();
+      EpilogEngraveProperty prop = new EpilogEngraveProperty();
```
[the remark this answers is not in the document]
Comment (buxtronix) [reply src/com/t_oster/liblasercut/drivers/EpilogCutter.java]: Done.

**Comment by buxtronix** [comment]
Squashed and pushed.

**Pull request merged**